docker-compose.yml

```diff
@@ -1,34 +1,59 @@
-postgresql:
-  restart: always
-  image: sameersbn/postgresql:9.4-18
-  environment:
+version: '2'
+
+networks:
```

Comment:
The networks key isn't needed because docker-compose will create an own network by default.